examples/image_classification_kornia.ipynb

```diff
@@ -0,0 +1,3856 @@
+{
```

Comment:
<p>Maybe we can move this under the <strong>Preprocessing the data</strong> section?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I feel like it makes sense here since the previous cells in this section since being able to map the id to the label feels like part of loading the dataset, and it doesn't actually get used in the preprocessing, no?</p>